data/build.gradle.kts

```diff
@@ -84,6 +84,9 @@ kotlin {
             languageSettings.apply {
                 optIn("kotlinx.coroutines.ExperimentalCoroutinesApi")
             }
+            dependencies {
+                implementation("com.google.truth:truth:1.0.1")
```

Comment:
Hey @anselmoalexandre For all dependencies, you need to add them to the `lib.versions file`. You can check the Readme on why we do this.

Great PR!

Reply:
> Hey @anselmoalexandre For all dependencies, you need to add them to the `lib.versions file`. You can check the Readme on why we do this.
> 
> Great PR!

🤦🏿‍♂️ my bad. Let me add it to version catalog

Reply:
> Hey @anselmoalexandre For all dependencies, you need to add them to the `lib.versions file`. You can check the Readme on why we do this.
> 
> Great PR!

Done. Added the dependency to `lib.versions` file